webui/react/src/components/kit/Form.module.scss

```diff
@@ -0,0 +1,3 @@
+.FormItem {
```

Comment:
super nit: I think we typically use camelCase instead of PascalCase for our classnames (https://khalilstemmler.com/blogs/camel-case-snake-case-pascal-case/)